detection-rules/impersonation_dmarc_failure_high_cred_phish_conclusion.yml

```
@@ -0,0 +1,16 @@
+name: "Impersonation: DMARC failure with high confidence credential theft intent"
+description: "Detects DMARC failures and messages with a high confidence of credential theft"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(distinct(headers.hops, .authentication_results.dmarc is not null), strings.ilike(.authentication_results.dmarc, "*fail"))
```

```suggestion
  and any(distinct(headers.hops, .authentication_results.dmarc is not null),
    strings.ilike(.authentication_results.dmarc, "*fail")
  )
```
wrapping my head around this (new line makes it a bit easier to read) - why do we need `distinct` here again?

---

In theory distinct will return the first occurrence, and since we have them in a relatively structured order we're assuming that that would be the authoritative answer.